src/lib/asn1/tests/TestASN1.cpp

```diff
@@ -362,6 +365,122 @@ static void TestASN1_ObjectID(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
 }
 
+static void TestASN1_FromTLVReader(nlTestSuite * inSuite, void * inContext)
+{
+    CHIP_ERROR err;
+    uint8_t tlvBuf[2048];
```

Comment:
does this run only on large devices (e.g. linux or darwin) or will this potentially run on smaller ones?

I believe small devices will not have sufficient stak for 6K buffers ... consider making these static.

Reply:
done